apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -652,4 +657,17 @@ defmodule OMG.Watcher.ExitProcessor do
 
     {invalidities_by_ife_id, state_db_updates}
   end
+
+  @spec add_scheduled_finalization_time(map()) :: map()
+  defp add_scheduled_finalization_time(%{eth_height: eth_height, call_data: %{utxo_pos: utxo_pos_enc}} = exit_event) do
+    {:utxo_position, blknum, _, _} = Utxo.Position.decode!(utxo_pos_enc)
+    {_block_hash, utxo_creation_block_timestamp} = RootChain.blocks(blknum)
+    {:ok, exit_block_timestamp} = Eth.get_block_timestamp_by_number(eth_height)
+    min_exit_period = Configuration.min_exit_period_seconds()
```

Comment:
if you need min exit period it needs to be in the process state, you cannot access configuration from here